usr/src/test/zfs-tests/tests/functional/cli_root/zpool_import/zpool_import_rename_001_pos.ksh

```diff
@@ -159,4 +161,13 @@ while (( i < ${#pools[*]} )); do
 	((i = i + 1))
 done
 
+VDEV_FILE=$(mktemp /tmp/tmp.XXXXXX)
+
+log_must truncate --size=128M $VDEV_FILE
+log_must $ZPOOL create testpool $VDEV_FILE
+log_must $ZFS create testpool/testfs
+log_must $ZPOOL export testpool
+ID=$($ZPOOL import | grep -A 1 "pool: testpool" | tail -n 1 | sed 's/.*id: //')
```

Comment:
This could be done a bit more cleaner, before exporting the pool:

ID=$($ZPOOL get -Ho value testpool)